pmd-packager/upgrade.sh

```diff
@@ -1,5 +1,7 @@
-VERSION=$(curl -s https://api.github.com/repos/pmd/pmd/releases/latest | jq --raw-output ".tag_name" | sed 's:.*/::')
+VERSION=$(curl -s https://api.github.com/repos/pmd/pmd/releases/latest | grep '.tag_name' | sed 's:.*/::' | sed 's:",::')
```

Comment:
We could use a single sed call like this:

```suggestion
VERSION="$(curl -s https://api.github.com/repos/pmd/pmd/releases/latest | sed -n '/"tag_name"/ s|.*\([0-9]\+\.[0-9]\+\.[0-9]\+\).*|\1| p')"
```